src/cf.data.pre

```diff
@@ -6481,6 +6482,11 @@ DOC_START
 		the HTTP standard. Enabling this feature could make you
 		liable for problems which it causes.
 
+		ignore-must-revalidate ignores any ``Cache-Control: must-revalidate``
+		headers received from a server. Doing this VIOLATES
+		the HTTP standard. Enabling this feature could make you
+		liable for problems which it causes.
+
```

Comment:
Please add text to warn the admin that enabling this feature prevents Squid from caching authenticated responses containing must-revalidate and no-cache directives.

Roughly speaking, folks enable ignore-must-revalidate to "cache more" rather than to "cache less". The opposite side effect (for authenticated responses) is likely to be unexpected by most admins. We should document that side effect.